pnpm-workspace.yaml

```diff
@@ -2,4 +2,4 @@ packages:
   - 'packages/*'
   - 'localenv/mock-account-servicing-entity'
   # exclude packages that are inside test directories
-  - '!**/test/**'
```

Comment:
Was not excluding anything from what I can tell (using `pnpm ls --depth -1 -r` to get packages). Removed to accommodate the `test/integration` workspace pkg but can rework this if it's actually doing something.

Reply:
this was probably a typo and should have been `tests`